instance/migrations/0114_auto_20190227_1534.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2019-02-27 15:34
+from __future__ import unicode_literals
+
+from django.db import migrations, models, transaction
+
+
+def forwards_func(apps, schema_editor):
```

Comment:
@giovannicimolin, wouldn't it be better to name the function based on what it actually does. Ditto for the reverse data migration function.